.github/workflows/charts-tests.yaml

```diff
@@ -50,32 +50,33 @@ jobs:
         name: Check if only documentation files changed
         run: ./scripts/check-docs-only-changes.sh ${{ github.event.pull_request.base.sha || 'HEAD~1' }} ${{ github.sha }}
 
-  generate:
-    timeout-minutes: 10
-    runs-on: ubuntu-latest
-    needs: [check-docs-only]
-    if: ${{ needs.check-docs-only.outputs.docs_only != 'true' }}
-    steps:
-      - name: Harden Runner
-        uses: step-security/harden-runner@ec9f2d5744a09debf3a187a3f4f675c53b671911 # v2.13.0
-        with:
-          egress-policy: audit
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-
-      - name: Setup go
-        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0
-        with:
-          go-version-file: go.mod
-
-      - uses: jdx/mise-action@c37c93293d6b742fc901e1406b8f764f6fb19dac # v2.4.4
-        with:
-          install: false
-
-      - name: Run manifests.charts
-        run: make manifests.charts
-
-      - name: Check generated files for diff
-        run: make verify.diff
+  # (TODO: re-enable after v2.0.0-alpha.3)
```

Comment:
Can we attach a more elaborate not here so that we know _why_ this is disabled?

Reply:
This is something we are going to fix today/tomorrow. Do we really need such an elaborate description?